Src/FluentAssertions/Execution/MessageBuilder.cs

```diff
@@ -91,7 +91,7 @@ private string FormatArgumentPlaceholders(string failureMessage, object[] failur
             string[] values = failureArgs.Select(a => Formatter.ToString(a, useLineBreaks)).ToArray();
             string formattedMessage = string.Format(failureMessage, values);
 
-            return formattedMessage.Replace("{{{{", "{{").Replace("}}}}", "}}");
+            return formattedMessage;
```

Comment:
You could directly return the result of the `Format` call

Reply:
If not too much of a problem I would rather leave it like it is. I like such an approach for
a) easier to debug, there is a place to take a look at the result before returning it
b) intention revealing name attached to the value

Reply:
Sure It was just a suggestion. By the way, I don't know about Visual Studio, but at least in Jetbrains Rider, you don't need to do this anymore. The debugger will show you all parts of an expression.